test/exe/Main.hs

```diff
@@ -1557,7 +1561,7 @@ completionTests
 outlineTests :: TestTree
 outlineTests = testGroup
   "outline"
-  [ testSessionWait "type class" $ do
+  [ testSession "type class" $ do
```

Comment:
Reminder to myself and other reviewers: This is a reasonable change since we don’t call `getDiagnostics`.